pkg/rebuild/schema/schema.go

```diff
@@ -229,6 +229,7 @@ type SmoketestAttempt struct {
 	Success           bool    `firestore:"success,omitempty"`
 	Message           string  `firestore:"message,omitempty"`
 	Strategy          string  `firestore:"strategy,omitempty"`
+	Dockerfile        string  `firestore:"dockerfile,omitempty"`
```

Comment:
wanna pull this out into a separate PR?

Reply:
Moved to https://github.com/wbxyz/oss-rebuild/tree/dockerfile-firestore